docs/advanced_topics/api/v2/extra_usage.rst

```diff
@@ -0,0 +1,86 @@
+=====================================
+Extra information on exposing the API
+=====================================
+
+Images
+------
+
+If you have in your model an image you can expose the image by using the ImageRenditionField.
+:class:`wagtail.images.api.fields.ImageRenditionField`. The ImageRenditionField will return url, width, height.
+
+..code-block:: python
+
+    class SomePage(Page):
+        ...
+        image = models.ForeignKey('wagtailimages.Image', null=True, blank=True, on_delete=models.SET_NULL, related_name='+')
+        ...
+
+        @property
+        def img(self):
+            return ImageRenditionField('width-700').to_representation(self.image)
+
+        @property
+        def alt(self):
+            return self.image.title
+
+        api_fields = [
+            ...
+            APIField('img'),
+            APIField('alt'),
+            ...
+        ]
+
+Alterantive you also can do the following
+
+..code-block:: python
+
+        api_field = [
+            APIField('image', serializer=ImageRenditionField('fill-300x300'))
+        ]
+
+If the name "image" is different as the can add the source name
+
+..code-block:: python
+    api_field = [
+            APIField('somename', serializer=ImageRenditionField('fill-300x300').source="myimage")
```

Comment:
I think this example has a typo

`APIField('image', serializer=ImageRenditionField('fill-300x300', source="promo_image"))`

the source=, should be passed in as a kwarg to the `ImageRenditionField` class

Also, this did not work though, still getting SourceImageIOError